microsoft/electionguard-ui
# ⛏️ Fix Key Selection Page and Upgrade to MUI 5

[//]: # (🚨 Please review the CONTRIBUTING.md in this repository. 💔Thank you!)

### Issue

Supports #87 

### Description

This fixes the key selection page in election setup and upgrades the package formerly known as Material UI, now rebranded to MUI.  It also upgrades all packages that were producing security warnings in npm.  The MUI upgrade is a major effort and isn't 100% complete yet.  There are still a couple of warnings, but it's a huge improvement and removes a number of runtime warnings that we had been receiving.  I'll fix the remaining upgrade issues (e.g. the `adaptV4Theme` shim in `theme.js`) in subsequent PR's.

### Testing
1. Run the app and log in
2. Click election setup
3. Click continue

Expected, something like this:

![image](https://user-images.githubusercontent.com/1769905/150216246-a108bef4-8dea-45e7-a24c-6e75e3832d92.png)

It should list keys (if they exist in the back-end), and they should be selectable.
